sydent/http/servlets/store_invite_servlet.py

```diff
@@ -168,15 +169,23 @@ def _redact(self, s, characters_to_reveal):
         :return: The redacted string.
         :rtype: unicode
         """
-        # If the string is shorter than the defined threshold, redact based on length
         if len(s) <= characters_to_reveal:
-            if len(s) > 5:
-                return s[:3] + u"..."
-            if len(s) > 1:
-                return s[0] + u"..."
-            return u"..."
-
-        # Otherwise truncate it and add an ellipses
+            if self.sydent.always_obfuscate:
+                # Always try to obfuscate *some* part of the string
+
+                # If the string is shorter than the defined threshold,
+                # redact based on length
+                if len(s) > 5:
+                    return s[:3] + u"..."
+                if len(s) > 1:
+                    return s[0] + u"..."
+                return u"..."
+
+            # Otherwise just return the original address
+            return s
+
+        # Truncate to the desired length. Note that this may not obfuscate the string
+        # if it already shorter than `characters_to_reveal`. Add an ellipses.
```

Comment:
If it is already shorter than `characters_to_reveal` won't we end up in the if-statement above? This comment confuses me.

Reply:
Yeah, I think the second sentence here was more relevant before I added the `return s` above. I've removed that bit, and cleaned up some of the other comments as well.